### DISCUSSION TITLE
Question: Can I disable automatic merge?

### DISCUSSION THREAD

**Discussion by TwentySeven28** [stashapp/stash#4502] [Q&A] [answered]
I have a lot of duplicates, and I find a lot of scenes are being automatically merged. When this happens, the Scene Duplicate Checker no longer finds these duplicates, and I cannot delete them easily.

I can find these duplicates in the Scenes view using the duplicated phash filter, but there is no way to delete these duplicates in bulk. Doing it one by one is taking a LONG time.

1. Can these merged scenes be split in bulk?
2. Could I delete all these identical merged videos in bulk?
3. Can I disable automatic merge so this doesn't keep happening?

Thanks!

**Comment by stg-annon** [comment]
1. Yes a plugin could do this in fact I think one exists 
2. plugin would likely also address this
3. I believe there might be an open issue for this, or that it was brought up in discord a few times


trouble with bulk delete operations is that there usually is still a preference the user has for which file to keep (likely the oldest one on exact duplicates) but this could change person to person

**Reply by TwentySeven28** [reply]
Thanks! I couldn’t find a plugin for this but I’ll look at the list again. Do you recall the name by any chance?

**Reply by Tscherno** [reply, marked as answer]
https://github.com/elkorol/Stash-Split-Duplicate-Merged-Files
